book/src/concepts/event-channel.md

```diff
@@ -173,7 +173,7 @@ and you also need to get read access:
 # }
 ```
 
-Then, in the `System`'s setup method:
+Then, in the `System`'s new method:
```

Comment:
should add backticks around this one -- it's obvious when it has it, but non-obvious when it doesn't